03.asynchronous/callback.js

```diff
@@ -0,0 +1,60 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+//エラーなし
+const db = new sqlite3.Database(":memory:");
+
+db.run("DROP TABLE IF EXISTS books", () => {
+  db.run(
+    "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, title VARCHAR(10) NOT NULL UNIQUE)",
+    () => {
+      db.run('INSERT INTO books(title) VALUES("チェリー本")', function () {
+        console.log(`ID${this.lastID}が追加されました`);
+        db.run(
+          'INSERT INTO books(title) VALUES("ブルーベリー本")',
+          function () {
+            console.log(`ID${this.lastID}が追加されました`);
+            db.all("SELECT * FROM books", (_error, rows) => {
+              rows.forEach((row) => console.log(row));
+              db.run("drop table if exists books");
```

Comment:
SQL のスタイルが他の部分と揃っていなさそうです。

```suggestion
              db.run("DROP TABLE IF EXISTS books");
```

また、この時点では `books` テーブルが存在していないといけないので、`IF EXISTS` 句をつけるのは適当ではなさそうです。

Reply:
SQLのスタイルを統一しました。https://github.com/SuzukaHori/js-practices/pull/3/commits/20290a8031219985b62904e9cc988162f52fe6f8
テーブルの削除処理で、なんとなく`IF EXISTS`をつけてしまっていました💦削除しました。https://github.com/SuzukaHori/js-practices/pull/3/commits/cf12108339ab021d371b7eaf007cca5dd58e5232